i3lock.c

```diff
@@ -138,6 +141,37 @@ static bool load_keymap(void) {
 }
 
 /*
+ * Loads the XKB compose table from the given locale.
+ *
+ */
+static bool load_compose_table(const char *locale) {
+    if (xkb_context == NULL) {
```

Comment:
Ok. I just use `load_keymap` as a template, because I never use the libxkbcommon.